blueprints/woo-shipping/woo-product.wxr

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Would the default WooCommerce product import be a better fit for this?

It has more products with all standard types (simple, variable) and would make testing of WooCommerce easier.

Reply:
@calvinrodrigues500 what do you think about using the default WooCommerce product import here?